tee-worker/ts-tests/post-checks/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "post-checks",
+  "version": "1.0.0",
+  "description": "",
+  "type": "module",
+  "main": "index.js",
+  "scripts": {
+    "start": "mocha --timeout 20000 --exit --sort -r ts-node/register --loader=ts-node/esm ./tests/**/*.test.ts"
+  },
+  "dependencies": {
+    "@litentry/enclave": "^4.0.0",
+    "@litentry/parachain-api": "file:../../client-api/parachain-api",
+    "@litentry/sidechain-api": "file:../../client-api/sidechain-api",
```

Comment:
I see that you have not added the generation steps for client-api in your yml file, so the deps here cannot be installed, I guess. see https://github.com/litentry/litentry-parachain/blob/dev/tee-worker/cli/lit_ts_integration_test.sh#L57-L68

Reply:
right! I forgot we don't track the generated files. I may just install it from NPM 🤔 
thank you, Verin!